ntttcp/ntttcp.TESTGEN.ps1

```diff
@@ -120,13 +122,25 @@ function banner {
 function test_ntttcp {
     [CmdletBinding()]
     Param(
-        [parameter(Mandatory=$true)] [String] $OutDir
+        [parameter(Mandatory=$true)] [String] $OutDir,
+        [parameter(Mandatory=$true)] [ValidateScript({Test-Path $_})] [String] $ConfigFile
     )
 
+    #Load the variables needed to generate the commands
     # execution time in seconds
     [int] $g_runtime = 10
     [int] $g_ptime   = 2
 
+    # execution time ($g_runtime) in seconds, wu, cd times ($g_ptime) will come from the Config ps1 file, if specified and take precedence over defaults 
+    Try
+    {
+        . .\$ConfigFile
+    }
+    Catch
+    {
+        Write-Host "$ConfigFile will not be used. Exception $($_.Exception.Message) in $($MyInvocation.MyCommand.Name)"
```

Comment:
Do we really want to continue to run the test if we failed to load the config file? I fear this sets us up for cases where we miss that something went wrong.

Reply:
@cmaloney-msft , we've initialized default values just before the try-catch block, so if the config file doesn't exist for some reason (which is highly unlikely anyway since it's bundled up as part of the repo), we fall back to default (current) behavior and generate commands for sampling